src/dsp_tools/utils/project_create.py

```diff
@@ -489,6 +490,10 @@ def _create_ontologies(
         project_remote: representation of the project on the DSP server
         verbose: verbose switch
 
+    Raises:
+        BaseError if an error occurs during the creation of an ontology. 
+        All other errors are printed, the process continues, but the success status will be false.
```

Comment:
This raises a question for me: Do all errors necessarily mean that the overall run was not successful? E.g. if the project already exists, but not the ontology, wouldn't there be an error because the project could not be created; but that doesn't have to mean that it was an unsuccessful run, if the user wanted to create the ontology and knows that the project already exists. What do you think?

Reply:
From testing perspective, it would count as unsuccessful.
And from user perpective: At the end, the following warning is printed:
```
f"WARNING: The project '{project_definition['project']['shortname']}' ({project_definition['project']['shortcode']}) "
              f"with its ontologies could be created, but during the creation process, some problems occurred. "
              f"Please carefully check the console output.")
```

The warning informs the user that some unexpexted things occured, and asks him to check the console output. I think that's a good compromise: We never know what the user expects, so I think that a gentle warning is a sound compromise.

Or should I reformulate the warning? Perhaps "unexpected things occurred" instead of "some problems occurred"?

Reply:
From a user perspective, this is absolutely ok, with this warning. But in terms of exit code (which may be relevant, if users have a sequence of instructions, which will not continue, if one of the commands has a non-zero exit code), it is a bit strange that when wanting to upload an ontology to an already existing project, I have to expect an error. It's probably not a big deal, but I still don't think it's ideal.